features/main/relation.feature

```diff
@@ -72,9 +72,7 @@ Feature: Relations support
       },
       "relatedToDummyFriend": [],
       "dummyBoolean": null,
-      "embeddedDummy": null,
-      "id": 1,
-      "symfony": "symfony",
+      "embeddedDummy": [],
       "age": null
```

Comment:
`id` and `symfony` properties where present twice, and it is execped to get an empty array on `embeddedDummy`